test/if.civet

```diff
@@ -296,15 +294,13 @@ describe "if", ->
       z
     } if w := g()
     ---
-    let ref;if (ref = g()) {
-    const w = ref;if (ref1 = f()) {
+    let ref;if (ref = g()) { const w = ref;if (ref1 = f()) {
```

Comment:
This seems to be missing a `let ref1`.

Reply:
It looks like it was missing that from the first time I added it, I'll see about adding it tomorrow.